File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(let i = 0; i < nums.length - 1; i++){
        for(let j = i + 1; j < nums.length; j++){
            const num = nums[i] + nums[j];
            
            if(num == target){
                return [i, j]
            }
        }
    }
    
    return null;
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        map = dict()
        for i in range(0, len(nums)):
            map[nums[i]] = i

        for i in range(0, len(nums)):
            rest = target - nums[i]
            
            if map.get(rest) != None and map.get(rest) != i :
                return [i, map.get(rest)]
        
        return []
            